mscore/inspector/inspector.cpp

```diff
@@ -86,7 +86,7 @@ void MuseScore::showInspector(bool visible)
             addDockWidget(Qt::RightDockWidgetArea, _inspector);
             }
       if (_inspector)
-            _inspector->setVisible(visible);
+            reDisplayDockWidget(_inspector, visible); // _inspector->setVisible(visible);
```

Comment:
better lose that comment, for history keeping we have git

Reply:
Thanks. Done. I will know to take more care on this count in the future :).